sentinel-dashboard/src/main/webapp/resources/app/scripts/services/auth_service.js

```diff
@@ -1,6 +1,13 @@
 var app = angular.module('sentinelDashboardApp');
 
 app.service('AuthService', ['$http', function ($http) {
+  this.version = function () {
```

Comment:
Can we put this API to another service like `VersionService`? It's actually not related to auth.

Reply:
Thanks for the guidance, that is more clever, I fix it, please review 🌹